freebasics/tests/test_views.py

```diff
@@ -0,0 +1,17 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase, Client
+
+
+class RegistrationViewTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_register_view_invalid_form(self):
+        # NOTE: empty form submission
+        response = self.client.post(reverse('user_register'), {
+            'username': 'testing',
+            'password': 'testing',
+        })
+        print response
+        self.assertRedirects(response, reverse('user_register'))
```

Comment:
shouldn't this be redirecting to `registration_done`
Please check that it's actually not just returning an error.